app/collections/create-form.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import apiClient from 'panoptes-client/lib/api-client';
+
+class CollectionsCreateForm extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
FYI, passing props into super isn't necessary if props aren't used in the constructor. Admittedly we've been inconsistent with this.